bower.json

```diff
@@ -9,7 +9,9 @@
     "angular-resource": "^1.4.0",
     "angular-route": "^1.4.0",
     "angular-sanitize": "^1.4.0",
-    "angular-touch": "^1.4.0"
+    "angular-touch": "^1.4.0",
+    "moment": "~2.13.0",
```

Comment:
I should be able to use it to create a nice date format correct? If so, i
will figure that out on implementing it.
On Jun 11, 2016 12:37 PM, "Julian Poyourow" <notifications@github.com>
wrote:

> In bower.json
> <https://github.com/jaylenw/AngularJsTodoApp/pull/10#discussion_r66710625>
> :
>
> > @@ -9,7 +9,9 @@
> >      "angular-resource": "^1.4.0",
> >      "angular-route": "^1.4.0",
> >      "angular-sanitize": "^1.4.0",
> > -    "angular-touch": "^1.4.0"
> > +    "angular-touch": "^1.4.0",
> > +    "moment": "~2.13.0",
>
> You've added momentJS to the project deps, but have never used it. Remove
> it or utilize it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jaylenw/AngularJsTodoApp/pull/10/files/7fbfd333bbfc497e22404dad679a95bf38ff251a#r66710625>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGNDs0fWr__VAZsoH4gomOjTI0tf9x0xks5qKw6SgaJpZM4Izf6h>
> .
>